documentation/plantuml/efgs_inbound.pu

```diff
@@ -0,0 +1,25 @@
+@startuml
+
+state INBOUND {
+    [*] --> STARTED
+    STARTED : Inbound triggered
+    STARTED --> VERIFY
+    VERIFY : Verify batch signature
+    VERIFY --> VALIDATE
+    VALIDATE : Validate batch data
+    VALIDATE --> STORE
+    STORE : Store keys to db
+    STORE --> FINISHED
+    STARTED --> ERROR
+    VERIFY --> FAIL
+    VALIDATE --> FAIL
+    ERROR --> FINISHED
+    ERROR --> ERROR
+    ERROR : Until max retries reached
```

Comment:
The keyword is missing. Would "Retry until max attempts reached" be too long? Same in the other pic.

Reply:
I think it's not. Changed that.